input/pagecontent/index.md

```diff
@@ -50,13 +50,34 @@ The RESTful interaction requirements for AU eRequesting actors, such as read and
 
 The diagram below illustrates the interactions between the various actors:
 
+<figure>
 <div> 
   <img src="au-erequesting-actor-interactions.svg" alt="FHIR RESTful read interactions" style="width:80%"/>
 </div>
+<figcaption>Figure 1: FHIR RESTful interactions</figcaption>
+</figure>
 <br/>
-*Figure 1: FHIR RESTful interactions*
+
```

Comment:
Just a comment on the general look and feel of this section now. 
- Could maybe introduce a sub-heading 1.5.1 Sequence of operations or similar to break the content down a bit?
- Could we make Figures 1 and 2 the same size, may be nicer to look at?